src/Sarif/Readers/SarifContractResolver.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Readers
 {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif")]
```

Comment:
This will occur often, so add an assembly-level suppression to GlobalSuppressions.cs to allow this name everywhere (and remove the individual suppressions). Not sure of the syntax. It might be:

```
[assembly: SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif")]
```

... so you'll want to research it.